packages/manager/src/features/components/PlansPanel/PlanSelection.tsx

```diff
@@ -106,12 +108,13 @@ export const PlanSelection = (props: PlanSelectionProps) => {
 
   const diskSize = selectedDiskSize ? selectedDiskSize : 0;
   const planTooSmall = diskSize > type.disk;
+  const isSamePlan = type.heading === currentPlanHeading;
   const tooltip = getToolTip({
-    sizeTooSmall: planTooSmall,
-    planIsDisabled: planIsDisabled,
     disabledToolTip: disabledToolTip,
+    isSamePlan: isSamePlan,
+    planIsDisabled: planIsDisabled,
+    sizeTooSmall: planTooSmall,
```

Comment:
We can shorthand this
```suggestion
    disabledToolTip,
    isSamePlan,
    planIsDisabled,
    sizeTooSmall: planTooSmall,
```